addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1168,6 +1168,27 @@ const ImagepickerUserValueWidget = UserValueWidget.extend({
         return _.extend({isVideo: this.isVideo}, this._super(...arguments));
     },
 
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * If parallax is enabled, remove it.
+     */
+    removeParallax: function () {
+        let $el = this.$target.find('> .s_parallax_bg');
+        if ($el.length) {
+            this.$target.removeClass('parallax s_parallax_is_fixed s_parallax_no_overflow_hidden');
+            this.$target.removeAttr("data-scroll-background-ratio");
+            this.trigger_up('option_update', {
+                optionNames: ['background', 'BackgroundPosition'],
+                name: 'target',
+                data: this.$target,
+            });
+            $el.remove();
+        }
+    },
```

Comment:
@bvr-odoo Please review all my explanations, as I said before: the UserValueWidgets *are not supposed to know about the target being modified*... They are not supposed to magically know the "s_parallax_bg" class and even less supposed to modify the DOM themself. The whole system rest on the fact that those DOM modifications are done by the snippet options special methods. Changing the DOM any other place than in those special methods will lead to race conditions in the system.